transports/transport-rest/transport-rest-client/src/main/java/org/apache/servicecomb/transport/rest/client/RestClientSenderFilter.java

```diff
@@ -38,6 +38,8 @@ public String getName() {
 
   @Override
   public CompletableFuture<Response> onFilter(Invocation invocation, FilterNode nextNode) {
+    invocation.onStartSendRequest();
```

Comment:
`invocation.getInvocationStageTrace().startSend()` is inside `RestClientSender.send`